src/Schema/Types/Scalars/DateTimeUtc.php

```diff
@@ -17,6 +17,6 @@ protected function format(Carbon $carbon): string
     protected function parse(mixed $value): Carbon
```

Comment:
Side-note about the comment in this class - it works with Carbon 2 and above right?

Reply:
Yes. It does, updated!